docs/rst/tutorials/core/qos/mutable/mutable.rst

```diff
@@ -0,0 +1,265 @@
+The value of the ownership strength is set from the value of the updated parameter.
+
+Configuration of initial QoS
+----------------------------
+
+Ownership Strength Policy is mutable, but Ownership Policy is not, so configuring EXCLUSIVE_OWNERSHIP_POLICY to all participants before running the ROS nodes is needed.
+To do that, inside the package, there are three xml files.
+Each one of them defines a profile for a publisher with a "large" ownership strength, another with a "small" ownership strength and a subscriber (that does not need an ownership strength definition).
+For the three of them, exclusive ownership is defined.
+
+.. tabs::
+
+    .. tab:: large_ownership_strength.xml
+
+        .. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/large_ownership_strength.xml
+            :language: xml
+
+    .. tab:: small_ownership_strength.xml
+
+        .. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/small_ownership_strength.xml
+            :language: xml
+
+    .. tab:: subscriber_exclusive_ownership.xml
+
+        .. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/subscriber_exclusive_ownership.xml
+            :language: xml
+
+
+Build
+-----
+
+Now the package is ready to be built.
+Change the directory to the workspace folder and build using colcon:
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+    cd ~/vulcanexus_ws
+    colcon build
+
+
+Run
+---
+
+Open three terminals in the workspace folder.
+On each of them, Vulcanexus installation, as well as the package installation is needed.
+Then, export the `FASTRTPS_DEFAULT_PROFILES_FILE` environment variable to point out to the corresponding profiles file and run the node.
+
+    * In the first terminal, run the `listener` node from the `demo_nodes_cpp`, configured with the `subscriber_exclusive_ownership.xml` file.
+
+    * | Then, in another terminal, run the first publisher, configured also with the `large_ownership_strength.xml` file.
+      | This Publisher will then be configured with ownership strength value of 10.
+      | At this point both nodes should be communicating, and the messages from Publisher 1 should be shown in the Subscriber.
+
+    * | In the third terminal, run the second publisher, configured with the `small_ownership_strength.xml` file.
+      | This Publisher will then be configured with ownership strength value of 2.
+      | This Publisher 2 starts sending messages (it can be seen that the number of the message starts from 1 while the messages from Publisher 1 are already in a higher number), and the Subscriber is still receiving messages from Publisher 1 and not from Publisher 2.
+      | This is because of the exclusive ownership.
```

Comment:
* In the first terminal, run the `listener` node from the `demo_nodes_cpp`, configured with the `subscriber_exclusive_ownership.xml` file.
    * Then, in another terminal, run the first publisher, configured also with the `large_ownership_strength.xml` file.
      This Publisher will then be configured with ownership strength value of 10.
      At this point both nodes should be communicating, and the messages from Publisher 1 should be shown in the Subscriber.
    * In the third terminal, run the second publisher, configured with the `small_ownership_strength.xml` file.
      This Publisher will then be configured with ownership strength value of 2.
      This Publisher 2 starts sending messages (it can be seen that the number of the message starts from 1 while the messages from Publisher 1 are already in a higher number), and the Subscriber is still receiving messages from Publisher 1 and not from Publisher 2.
      This is because of the exclusive ownership.

Reply:
Done